README.md

```diff
@@ -6,7 +6,11 @@ This repository contains Flutter sample code demonstrating the capabilities of A
 
 Visit [earlyadopter.esri.com](https://earlyadopter.esri.com/) and download the ArcGIS Maps SDK for Flutter package. Follow the instructions to unpack it.
 
-Clone or download this repository into the same parent directory as the `arcgis_maps_package`. Your file structure should be:
+Clone or download this repository into the same parent directory as the `arcgis_maps_package`.
+
+Note on Windows: This repository uses symlinks. Git must be configured to use symlinks and have permission to create symlinks. To configure git, run `git config --system core.symlinks true`. To have permission to create symlinks, either clone the repository in an elevated "Administrator" command prompt, or go to "Settings > For developers" and turn on "Developer Mode".
```

Comment:
```suggestion
Note on Windows: This repository uses symlinks. Git must be configured to use symlinks and have permission to create symlinks. To configure git, run `git config --system core.symlinks true`. To have permission to create symlinks, either clone the repository in an elevated "Administrator" command prompt, or go to "Settings > Update & Security > For developers" and turn on "Developer Mode".
```
Unless it's different for you, in which case maybe we make it more generic like 'search for the "For developers" section in Settings and turn on "Developer Mode"'.

Same applies to 'Note on Windows' below.

Reply:
Fixed x 2